core/templates/pages/exploration-player-page/services/question-player-engine.service.ts

```diff
@@ -94,6 +87,7 @@ export class QuestionPlayerEngineService {
       successCallback: (initialCard: StateCard, nextFocusLabel: string) => void,
       errorCallback: () => void): void {
     if (!this.questions || this.questions.length === 0) {
+      this.alertsService.addWarning('Questions can not be empty.');
```

Comment:
"There are no questions to display."

(The end user has no idea that there is a variable called 'questions', and the phrasing "Questions cannot be empty" does not make sense in plain English.)

Reply:
done.